recipes/physlr/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/bash
set -eu
```
Fail on error or undefined variable.